src/main/java/TicTacToe/Display.java

```diff
@@ -0,0 +1,65 @@
+package TicTacToe;
+
+import java.io.*;
+import java.util.List;
+
+public class Display {
+    public OutputStream outputStream;
+    private PrintStream printStream;
+    private BufferedReader inputStream;
+
+    public Display(PrintStream outputStream, InputStream inputStream) {
+        this.outputStream = outputStream;
+        this.printStream = new PrintStream(outputStream);
+        this.inputStream = new BufferedReader(new InputStreamReader(inputStream));
+    }
+
+    public void printGreeting() {
+        printStream.println("Welcome to TicTacToe");
+    }
+
+    public void printBoard(Board board) {
+        List<String> grid = board.getGridAsList();
+
+        String string = grid.get(0) + " | " + grid.get(1) + " | " + grid.get(2) + "\n"
+                + grid.get(3) + " | " + grid.get(4) + " | " + grid.get(5) + "\n"
+                + grid.get(6) + " | " + grid.get(7) + " | " + grid.get(8) + "\n";
+
+        printStream.println(string);
+    }
+
+    public void announceWinner(String mark) {
+        printStream.println("Player " + mark + " wins!");
+    }
+
+    public void announceDraw() {
+        printStream.println("Draw!");
+    }
+
+    public void announcePlayerTurn(String mark) {
```

Comment:
I can see kind of a pattern in your method with `announce...` maybe you could extract them in an `Announcement` class